docker/edge/http/script.sh

```diff
@@ -45,41 +43,46 @@ do
    fi
 done
 
+#cat generate_token.txt
+
 token=`grep "X-Subject-Token" generate_token.txt | cut -f 2 -d ":" |  sed 's/\r$//g' | tr -d '\n'`
 echo -----------------------------
 echo IDM token is $token 
 echo -----------------------------
 
-#Register application, it will return Application ID and Password. ID and Password will use to genenrate access token
+#Fetch Application ID and Password. ID and Password will use to genenrate access token
 for connect1 in 1 2 3
 do
-  curl -sb --include --request POST  -H "Content-Type: application/json"  -H "X-Auth-token: $token" --data-binary "{
-    \"application\": {
-      \"name\": \"$appName\",
-      \"description\": \"description\",
-      \"redirect_uri\": \"$callbackUrl\",
-      \"url\": \"$appUri\",
-      \"grant_type\": [
-        \"authorization_code\",
-        \"implicit\",
-        \"password\"
-      ],
-      \"token_types\": [
-          \"jwt\",
-          \"permanent\"
-      ]
-    }
-  }" \
-  "http://$IP_ADDRESS:3000/v1/applications" > accessAPI.txt
+  curl --include \
+     --request GET \
+     --header "X-Auth-token: $token" \
+  "http://$IP_ADDRESS:3000/v1/applications" > API_detail.js
   if [ $? -eq 0 ]; then
      break
   else
      echo reconnecting...
      continue
-   fi
+  fi
+done
+
+ID=`cat API_detail.js | tail -1| jq -r '.[] | .[] | .id' | tr -d '\n'`
+
+
+#To fetch secret of the Application 
+for connect2 in 1 2 3 
+do 
+  curl --include \
+     --header "X-Auth-token: $token" \
+  "http://$IP_ADDRESS:3000/v1/applications/$ID" > accessAPI.txt
```

Comment:
[Suggestion] accessAPI.txt is a hidden file better to make it hidden file.

Reply:
Comment Incorporated